pkg/controller/servicemesh/webhooks/mutation/common_test.go

```diff
@@ -14,14 +15,21 @@ import (
 
 	"github.com/maistra/istio-operator/pkg/controller/common"
 	"github.com/maistra/istio-operator/pkg/controller/common/test"
+	"github.com/maistra/istio-operator/pkg/controller/versions"
 )
 
 var (
 	ctx        = common.NewContextWithLog(context.Background(), logf.Log)
 	testScheme = test.GetScheme()
 )
 
-var acceptWithNoMutation = admission.Allowed("")
+func getDefaultAdmissionResponse() admission.Response {
```

Comment:
I think instead of this func we should change the tests to include the expected mutations for every test case. This `getDefaultAdmissionReponse` only works for test cases that don't touch `openshiftRoute.enabled` (which none of them currently do) but it's not really generic enough to warrant the name. Just supplying a spec that explicitly enables openshiftRoute will break this.

We will also adjust the tests properly, apparently the `no mutation 2` test case did actually have mutations on the profile (and now also on the openshiftRoute.enabled field) so I guess the name isn't appropriate anymore. We can add new test cases with no mutation